docs/developer-docs/defi/tokens/token-standards.mdx

```diff
@@ -0,0 +1,197 @@
+
+[ICRC](https://github.com/dfinity/ICRC) stands for `Internet Computer Request for Comments'. ICRC standards are proposed by the community working group and are not specific to tokens; they can be created for anything on ICP, such as services, canister calls, or authentication methods. However, this documentation will detail the ICRC standards that are specific to tokens.
+
+In the past, the DIP-721 standard was promoted and used by projects on the network for deploying non-fungible tokens. The ICRC-7 standard has since been developed as a newer alternative to the DIP-721 standard.
+
+## ICP tokens
+
+The ICP token is the network's native token used for various utility and governance functions, such as:
+
+- Paying for a canister's resources by being converted into cycles.
+
+- Rewarding node providers for contributing resources to the network.
+
+- Rewarding neurons for staking ICP into the neuron and voting on governance proposals.
+
+The ICP token is similar to Bitcoin in the following ways:
+
+-   Each ICP token is divisible 10^8 times.
+
+-   All transactions are stored in the ledger, starting with the genesis initial state.
+
+-   Tokens are entirely fungible.
+
+-   `AccountIdentifier`s are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
+
+The ICP token differs from Bitcoin in the following ways:
+
+-   Rather than using proof of work, staked participant nodes use a variant of threshold BLS signatures to agree on a valid state of the chain.
+
+-   Any transaction can store an 8-byte memo; this memo field is used by the Rosetta API to store the nonce that distinguishes between transactions. However, other uses for the field are possible.
+
+View an example dapp for [ICP transfers](https://github.com/dfinity/examples/tree/master/motoko/icp_transfer).
+
+## ICRC-1
+
+The ICRC-1 standard is for fungible tokens on ICP. It uses an `Account` value that corresponds to the pair `(principal, subaccount)`, where a principal can have multiple accounts, and each subaccount of a principal is identified by a 32-byte string.
+
+The ICRC-1 standard is considered the 'base' standard and intentionally excludes certain features and functions, such as:
+
+The base standard intentionally excludes some ledger functions, such as:
+
+- Reliable transaction notifications for smart contracts.
+
+- A block structure and the interface for fetching blocks.
+
+- Pre-signed transactions.
+
+ICRC-1 supports extensions of this standard to implement these missing functions, and the `icrc1_supported_standards` endpoint can be queried to return the name of all supported extensions.
+
+### Token metadata
+
+Available metadata entries for ICRC-1 tokens include:
```

Comment:
It is unclear what is metadata and why it is needed.

Reply:
Added context of what metadata is